rslib/src/decks/tree.rs

```diff
@@ -216,20 +216,17 @@ fn sum_counts_and_apply_limits_v3(
     for child in &mut node.children {
         individually_capped_total += sum_counts_and_apply_limits_v3(child, limits);
     }
+    // so we can calculate this total
     node.total_including_children = individually_capped_total.total;
```

Comment:
Does this field serve any purpose? I couldn't find any other reference to it, and don't know value it is supposed to have with the new approach.
If we could drop it, this code could be simplified.

Reply:
When the deck list screen is refactored, I was thinking it would be nice to have a hover tooltip like we do with the graphs. "Why am I not seeing my cards?" is one of the most common questions, and if the tooltip showed the total cards in the deck with and without children/limits, I thought that might make it easier for users to understand what is going on. The numbers are supposed to be exported via decks.proto:DeckTreeNode, but not yet used by the UI. Would a tally of the child deck's totals without limits be easier?

Reply:
I don't think either is particularly complicated to implement, but I don't quite understand the usefulness of the first alternative.
Let's say the user has clicked on deck A with a limit of 100. It contains 200 eligible cards, but due to descendant limits it only shows 50.

- Alternative 1: Count all cards, only considering the respective source deck's limit. This may be any number between 50 and 200.
- Alternative 2: Count all cards, only considering deck A's limit. This is a number between 50 and 100.

Have I got this right? If so, I'd say 2 is more useful, because it explains why there are less cards than the clicked-on deck's limit would allow for.

Reply:
From decks.proto:

  // card counts without children or limits applied
  uint32 intraday_learning = 9;
  uint32 interday_learning_uncapped = 10;
  uint32 new_uncapped = 11;
  uint32 review_uncapped = 12;

These would allow the user to see how many cards may be in that particular deck beyond the set limits

  uint32 total_in_deck = 13;
  uint32 total_including_children = 14;

These were so the user could see the entire number of cards in the deck, regardless of if they are due or not. Not sure it would make sense to apply limits there? The UI could show "total cards in this deck" and total_including_children-total as "total cards in child decks" for example. These two are not so much about answering the daily limits questions, but for things like when cards are due on a different day and users wonder where they have gone.

Reply:
> The UI could show "total cards in this deck" and total_including_children-total as "total cards in child decks" for example. These two are not so much about answering the daily limits questions, but for things like when cards are due on a different day and users wonder where they have gone.

Ah, this also makes sense.